### PR TITLE
DSD-506: Chakra StatusBadge

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-506](https://jira.nypl.org/browse/DSD-506)

## This PR does the following:
- Rebuilds the `StatusBadge` component with Chakra.

## How has this been tested?

Through Storybook.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr709-mny4cueqvxzenshu8oxymhwutbs9wjsd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6156162bcc9cc41fc35b0680

File: src/components/StatusBadge/StatusBadge.tsx [hunk 1]
```diff
@@ -1,37 +1,30 @@
...
-  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
-  blockName?: string;
-  /** ClassName that appears in addition to "status-badge" */
+  /** Additional class for the component */
   className?: string;
-  /** Level of badge **/
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+  /** Level of the status badge */
   level?: "low" | "medium" | "high";
```
Comment (bigfishdesign13) [inline src/components/StatusBadge/StatusBadge.tsx]: Why this and not an `enum` like we use elsewhere?

**Review by bigfishdesign13 — COMMENTED** [review]
This looks good.  I just have one question about how the variant names are handled.

File: src/components/StatusBadge/StatusBadge.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/StatusBadge/StatusBadge.tsx]: You're right. will update.

**Comment by EdwinGuzman** [comment]
Updated.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr709-mny4cueqvxzenshu8oxymhwutbs9wjsd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6156162bcc9cc41fc35b0680

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr709-mny4cueqvxzenshu8oxymhwutbs9wjsd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6156162bcc9cc41fc35b0680

**Pull request merged**